src/content/configuration/resolve.mdx

```diff
@@ -193,6 +193,52 @@ module.exports = {
 };
 ```
 
+Webpack will match export conditions that is listed within the `resolve.conditionNames` array.
+
+The key order in the `exports` field is significant. During condition matching, earlier entries have higher priority and take precedence over later entries.
```

Comment:
Do you mean `conditionNames` field here? `exports` is an object which I believe has no order in JavaScript.

Reply:
i thought so too in the beginning, but no, i meant the `exports` object.

see: https://nodejs.org/api/packages.html#conditional-exports